packages/admin/cms-admin/src/contentScope/ContentScopeIndicator.tsx

```diff
@@ -75,6 +76,17 @@ const ScopeIndicator = styled("div", { shouldForwardProp: (prop) => prop !== "gl
     font-weight: ${({ global }) => (global ? 600 : 400)};
     line-height: 16px;
     text-transform: ${({ global }) => (global ? "uppercase" : "none")};
+
+    ${({ theme }) => theme.breakpoints.up("sm")} {
+        width: auto;
```

Comment:
What `width` is being overridden here? 🤔

Reply:
This should probably unset the max-width in L65.